tests/kubic/repositories.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Configure Kubic repositories
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils "zypper_call";
+
+sub run {
+    if (get_var('MIRROR_HTTP')) {
```

Comment:
What about this https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/caasp/first_boot.pm#L30
You copied code, please remove original.

Reply:
Good idea. I've removed it.